src/streaming/ManifestLoader.js

```diff
@@ -155,6 +159,13 @@ function ManifestLoader(config) {
                         manifest.originalUrl = manifest.url;
                     }
 
+                    // In the following, we only use the first Location entry even if many are available
+                    // Compare with ManifestUpdater/DashManifestModel
+                    if (manifest.hasOwnProperty('Location')) {
+                        baseUri = urlUtils.parseBaseUrl(manifest.manifest.Location_asArray[0]);
```

Comment:
@TobbeEdgeware, is there too much manifest ? I have an issue when I try to play the stream :

http://wowzaec2demo.streamlock.net/live/bigbuckbunny-enc-wv.stream/manifest_mvtime.mpd